### PR TITLE
Update cloud-recording.md

### PR DESCRIPTION
**Pull request by cchiffersMSFT**
Adding additional FAQ's after discussion with Teams Engineering.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/cloud-recording.md [hunk 1]
```diff
@@ -335,6 +335,14 @@ Everyone with view access will see a red icon next to the file in your OneDrive
   
 The file owner will receive an email notification when the TMR expires and will be directed to the recycle bin to recover the TMR if they desire to do so.
   
+**What happens if I move a recording to another location in OneDrive/SharePoint?**
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:338]:
```suggestion
**What happens if I move a recording to another location in OneDrive or SharePoint?**
```

File: Teams/cloud-recording.md [hunk 2]
```diff
@@ -335,6 +335,14 @@ Everyone with view access will see a red icon next to the file in your OneDrive
...
+  
+TMR expiration tags follows the recording. On the expiration date, the recording will be moved to the OneDrive or SharePoint recycle bin. Users can choose to download the recording if they wish to save it, or anyone who has edit and delete permissions can modify and extend the expiration date.
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:340]:
```suggestion
TMR expiration tags follow the recording. On the expiration date, the recording will be moved to the OneDrive or SharePoint recycle bin. Users can choose to download the recording if they wish to save it, or anyone who has edit and delete permissions can modify and extend the expiration date.
```

File: Teams/cloud-recording.md [hunk 3]
```diff
@@ -335,6 +335,14 @@ Everyone with view access will see a red icon next to the file in your OneDrive
...
+  
+**Will changes to expiration policies affect existing meeting recordings?**
+  
+No. Changes to expiration policies do not affect existing meeting recordings. Changing expiration settings will only impact newly created TMRs from that point forwards.
+  
 **What SKUs are required for this feature?**
   
 All SKUs will have this feature by default. A1 users will be defaulted to a 30-day expiration period and won’t be able to modify the expiration date
```
Comment (JohanFreelancer9) [inline Teams/cloud-recording.md:348]:
```suggestion
All SKUs will have this feature by default. A1 users will be defaulted to a 30-day expiration period and won’t be able to modify the expiration date.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @cchiffersMSFT, kindly commit the minor suggested changes. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by scanum** [comment]
@tonysmit Hope all is well. Wanted to follow up on this PR. Thanks.

**Comment by scanum** [comment]
@tonysmit PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by cchiffersMSFT** [comment]
Unsure where the conflict was. Have re-submitted.

**Comment by scanum** [comment]
@CarolynRowe 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
This article is undergoing extensive changes by another writer.  I'm going to close this PR, but send the content to the writer to review and consider adding.  Also, the PR was submitted to me with Warnings.

**Pull request closed without merging**